qa/rpc-tests/excessive.py

```diff
@@ -74,15 +74,6 @@ def setup_network(self, split=False):
                 pdb.set_trace()
 
     def run_test(self):
-        # Temporary workaround for the may15th hardfork. These tests will not work if the fork is enabled
-        # because we are forcing an EB of 32MB after each block is mined which invalidates many of these tests.
-        # So we set the mocktime to sometime far in advance of the fork.
-        MAY152018_START_TIME = 1526400000;
-        self.nodes[0].setmocktime(MAY152018_START_TIME - 10000)
-        self.nodes[1].setmocktime(MAY152018_START_TIME - 10000)
```

Comment:
A `for` loop saves two lines of code here...

Reply:
how so? we removed this 6 lines.

Reply:
Ooops. Yeah, I guess I am colorblind :D